singolo/js/script.js

```diff
@@ -0,0 +1,253 @@
+const navigation = document.querySelector(".navigation");
+const header__navigation = document.querySelector(".header__navigation");
+const header_content = document.querySelector(".header-content");
+const burger = document.querySelector(".burger-menu");
+const body = document.querySelector("body");
+const bod = document.querySelector(".services");
+const bodyChildren = body.children;
+const nav = document.querySelectorAll(".navigation__link");
+const all = document.querySelector(".all");
+const web = document.querySelector(".web-design");
+const graphic = document.querySelector(".graphic-design");
+const artwork = document.querySelector(".artwork");
+const button_filter = document.querySelectorAll(".button-filter");
+const images = document.querySelectorAll(".images");
+let numb = [];
+let remove_active_button = button_filter[0];
+let remove = nav[0];
+
+//OPEN THE BURGER MENU
+burger.addEventListener("click", () => {
+  burger.classList.toggle("click");
+  header_content.classList.toggle("open");
+
+});
+
+//SCROLL TRACKING
+for (let i = 0; i < nav.length; i++) {
+  window.addEventListener("scroll", () => {
+    console.log(bodyChildren[3].offsetTop);
+    if (
```

Comment:
the block _if-else_ is repeated three times

Reply:
Fixed